lib/components/Attachments/AttachmentProgressCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { File } from "neetoicons";
+import { Typography } from "neetoui";
+
+const AttachmentProgressCard = ({ attachment }) => {
```

Comment:
Create an `Attachment` folder. The `Card` component would be the index. Rename this component as `Progress`.